README.md

```diff
@@ -105,3 +105,10 @@ As a minimum, any modules recommended as part of this reference architecture sho
 ## Contributing
 
 To Contribute to this project, please see the [Contributing Guide](./CONTRIBUTING.md).
+
+## Contributors
+
+* Dominic Harries - Dev Lead at IBM Garage for Cloud
```

Comment:
```suggestion
* Bethany Griggs - Sr. Software Engineer - Red Hat
* Dominic Harries - Dev Lead at IBM Garage for Cloud
```

(assuming we're going alphabetical by first name :))

Reply:
@BethGriggs I was planning on alphabetical